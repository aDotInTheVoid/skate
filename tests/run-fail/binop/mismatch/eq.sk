fn main() {
    null == 2;
}