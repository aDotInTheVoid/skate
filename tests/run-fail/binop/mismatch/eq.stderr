error: Unknown binop `==`, for `null` and `int`
  ┌─ $DD/tests/run-fail/binop/mismatch/eq.sk:2:10
  │
2 │     null == 2;
  │     ---- ^^ - LHS evaluated to 2
  │     │    │   
  │     │    In this operator
  │     LHS evaluated to null

